ganga/GangaTest/Framework/runner.py

```diff
@@ -55,7 +55,7 @@
 # search for local tests lookup
 myConfig.addOption('SearchLocalTests', True, 'search for local tests lookup')
 # search for tests packaged in release (PACKAGE/old_test dir) lookup
-myConfig.addOption('SearchReleaseTests', True, 'search for tests packaged in release (PACKAGE/old_test dir) lookup')
```

Comment:
Do we know what this config option does - are we sure it wants to be deleted?

Reply:
It is never used as it relates to the old testing framework.